src/commands/developers/EvalCommand.ts

```diff
@@ -56,8 +56,7 @@ export class EvalCommand extends BaseCommand {
             const cleaned = this.clean(String(e));
             const isTooLong = cleaned.length > 1024;
             const error = isTooLong ? `${await this.hastebin(cleaned)}.js` : `\`\`\`js\n${cleaned}\`\`\``;
-
-            embed.setColor("RED").addFields([{ name: i18n.__("commands.developers.eval.errorString"), value: error }]);
+            embed.setColor("Red").addFields([{ name: i18n.__("commands.developers.eval.errorString"), value: error }]);
```

Comment:
The blank line that was here shouldn't be deleted. Variable declaration should be separated from function/method calls.